src/Service.Tests/SqlTests/GraphQLMutationTests/MsSqlGraphQLMutationTests.cs

```diff
@@ -58,6 +58,29 @@ public async Task InsertMutation()
             await InsertMutation(msSqlQuery);
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new Publisher with name = 'New publisher'
+        /// <code>Check: </code> Mutation fails because the database policy (@item.name ne 'New publisher') prohibits insertion of records with name = 'New publisher'.
+        /// </summary>
+        [TestMethod]
+        public async Task InsertMutationFailingDatabasePolicy()
+        {
+            string errorMessage = "Could not insert row with given values.";
```

Comment:
Do we check for this errorMessage anywhere?

Reply:
here: https://github.com/Azure/data-api-builder/blob/b8a16017517734823898ce2c122e14aba3fc92a8/src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs#L64